README.md

```diff
@@ -1,2 +1,1610 @@
+[main]     Selecting previously unselected package libc-dev-bin.
+[main]     Preparing to unpack .../06-libc-dev-bin_2.28-10+rpi1_armhf.deb ...
+[main]     Unpacking libc-dev-bin (2.28-10+rpi1) ...
+[main]     Selecting previously unselected package linux-libc-dev:armhf.
+[main]     Preparing to unpack .../07-linux-libc-dev_4.19.67-2+deb10u1_armhf.deb ...
+[main]     Unpacking linux-libc-dev:armhf (4.19.67-2+deb10u1) ...
+[main]     Selecting previously unselected package libc6-dev:armhf.
+[main]     Preparing to unpack .../08-libc6-dev_2.28-10+rpi1_armhf.deb ...
+[main]     Unpacking libc6-dev:armhf (2.28-10+rpi1) ...
+[main]     Selecting previously unselected package libisl19:armhf.
+[main]     Preparing to unpack .../09-libisl19_0.20-2_armhf.deb ...
+[main]     Unpacking libisl19:armhf (0.20-2) ...
+[main]     Selecting previously unselected package libmpfr6:armhf.
+[main]     Preparing to unpack .../10-libmpfr6_4.0.2-1_armhf.deb ...
+[main]     Unpacking libmpfr6:armhf (4.0.2-1) ...
+[main]     Selecting previously unselected package libmpc3:armhf.
+[main]     Preparing to unpack .../11-libmpc3_1.1.0-1_armhf.deb ...
+[main]     Unpacking libmpc3:armhf (1.1.0-1) ...
+[main]     Selecting previously unselected package cpp-8.
+[main]     Preparing to unpack .../12-cpp-8_8.3.0-6+rpi1_armhf.deb ...
+[main]     Unpacking cpp-8 (8.3.0-6+rpi1) ...
+[main]     Selecting previously unselected package cpp.
+[main]     Preparing to unpack .../13-cpp_4%3a8.3.0-1+rpi2_armhf.deb ...
+[main]     Unpacking cpp (4:8.3.0-1+rpi2) ...
+[main]     Selecting previously unselected package libcc1-0:armhf.
+[main]     Preparing to unpack .../14-libcc1-0_8.3.0-6+rpi1_armhf.deb ...
+[main]     Unpacking libcc1-0:armhf (8.3.0-6+rpi1) ...
+[main]     Selecting previously unselected package libgomp1:armhf.
+[main]     Preparing to unpack .../15-libgomp1_8.3.0-6+rpi1_armhf.deb ...
+[main]     Unpacking libgomp1:armhf (8.3.0-6+rpi1) ...
+[main]     Selecting previously unselected package libatomic1:armhf.
+[main]     Preparing to unpack .../16-libatomic1_8.3.0-6+rpi1_armhf.deb ...
+[main]     Unpacking libatomic1:armhf (8.3.0-6+rpi1) ...
+[main]     Selecting previously unselected package libasan5:armhf.
+[main]     Preparing to unpack .../17-libasan5_8.3.0-6+rpi1_armhf.deb ...
+[main]     Unpacking libasan5:armhf (8.3.0-6+rpi1) ...
+[main]     Selecting previously unselected package libubsan1:armhf.
+[main]     Preparing to unpack .../18-libubsan1_8.3.0-6+rpi1_armhf.deb ...
+[main]     Unpacking libubsan1:armhf (8.3.0-6+rpi1) ...
+[main]     Selecting previously unselected package libgcc-8-dev:armhf.
+[main]     Preparing to unpack .../19-libgcc-8-dev_8.3.0-6+rpi1_armhf.deb ...
+[main]     Unpacking libgcc-8-dev:armhf (8.3.0-6+rpi1) ...
+[main]     Selecting previously unselected package gcc-8.
+[main]     Preparing to unpack .../20-gcc-8_8.3.0-6+rpi1_armhf.deb ...
+[main]     Unpacking gcc-8 (8.3.0-6+rpi1) ...
+[main]     Selec
```

Comment:
can be removed, i suppose

Reply:
I'd rather stick with the complete build log as it shows just how much stuff was required for install.